frontend/src/App.vue

```diff
@@ -61,9 +61,29 @@ export default {
       const itemFqid = contextualItem?.item?.fqid;
       await this.rpc.invoke("performAction", [itemFqid, index, contextualItem.context]);
     },
+
+    getItemByFqid(fqid) {
+      let items = [];
+      this.collections.forEach(col => {
+        items = items.concat(col.items);
+      });
+      const flatten = [];
+      let itemnode;
+      while (items.length > 0) {
+        itemnode = items.shift();
+        flatten.push(itemnode);
+        if (!itemnode.items) continue;
+        itemnode.items.forEach(item => {
```

Comment:
If the spread operator `...` is supported in this vuejs configuration we can simplify this logic

instead of:

```javascript
        itemnode.items.forEach(item => {
          items.push(item);
        });
```

We simply write:

```javascript
          items.push(...itemNode.items);
```

This works because the `Array.push` accepts a variable number of arguments.
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push